src/Hammer.js

```diff
@@ -90,34 +91,33 @@ function updateHammer (hammer, props) {
 	});
 }
 
-var HammerComponent = React.createClass({
+class HammerComponent extends React.Component {
```

Comment:
Done, are there any official tests for hammer, or some good place to test this change?